libeos-payg/clock-jump-source.h

```diff
@@ -0,0 +1,36 @@
+/* -*- mode: C; c-file-style: "gnu"; indent-tabs-mode: nil; -*-
+ *
+ * Copyright © 2019 Endless Mobile, Inc.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Alternatively, the contents of this file may be used under the terms of the
+ * GNU Lesser General Public License Version 2.1 or later (the "LGPL"), in
+ * which case the provisions of the LGPL are applicable instead of those above.
+ * If you wish to allow use of your version of this file only under the terms
+ * of the LGPL, and not to allow others to use your version of this file under
+ * the terms of the MPL, indicate your decision by deleting the provisions
+ * above and replace them with the notice and other provisions required by the
+ * LGPL. If you do not delete the provisions above, a recipient may use your
+ * version of this file under the terms of either the MPL or the LGPL.
+ */
+
+#pragma once
+
+#include <glib.h>
+
+G_BEGIN_DECLS
+
+#if SIZEOF_TIME_T == 8
+#  define TIME_MAX G_MAXINT64
+#elif SIZEOF_TIME_T == 4
+#  define TIME_MAX G_MAXINT32
```

Comment:
Maybe add a `#warning Code is not Y2038 compliant` here?

Reply:
Or `#error`, as per my other Y2038 comment

Reply:
This can't be `#error` because we do compile and ship `eos-payg` on ARM.

I don't really see the advantage of a `#warning` either because there's likely to be no action we can take for *years*, and no-one checks compile warnings on ARM anyway.